index.html

```diff
@@ -1,20 +1,154 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
 
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
-    <link rel="stylesheet" type="text/css" href="css/grid.css">
+    <link rel="stylesheet" type="text/css" href="../css/grid.css" />
+    <link rel="stylesheet" type="text/css" href="../css/main.css" />
+
+    <link
+      rel="stylesheet"
+      href="https://fonts.googleapis.com/css?family=Lemonada"
+    />
+    <link
+      href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css"
+      rel="stylesheet"
+    />
     <!-- Add a link to your custom CSS here -->
 
     <title>Responsive grid project</title>
-</head>
-
-<body>
+  </head>
 
+  <body>
     <!-- Add your markup here -->
 
-</body>
+    <header>
+      <nav class="container nav-bar">
+        <div class="row head-logo">
+          <h3 class="logo">sumo</h3>
+        </div>
+
+        <ul class="row nav-list">
+          <li class="sm-col-4 md-col-4 lg-col-4">
```

Comment:
Always start from `col-N`, which is equivalent to `xs-col-N`. Otherwise you will be missing any styling for small screens (e.g. smartphones). 

Additionally, you only need to include larger-screen classes if they change. Here's a few examples:
```html
<li class="col-4">            <!-- col-4 for all screen sizes -->
<li class="xs-col-4">         <!-- same as the one above -->
<li class="col-12 md-col-6">  <!-- full-width on small screens, half-width on medium-size and above -->
```